config/_shared.json

```diff
@@ -20,6 +20,7 @@
 	"google_analytics_key": false,
 	"google_adwords_conversion_id": false,
 	"google_adwords_conversion_id_jetpack": false,
+	"google_maps_api_key": "ABC",
```

Comment:
I'd expect that we don't have a shared API key for all envs (so this one would be `false`), but then specify a different key for each of the environments.